<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScreenshotDeviceSelectionDialog</class>
 <widget class="QDialog" name="ScreenshotDeviceSelectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="ipLabel">
      <property name="text">
       <string>Device IP</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ipLineEdit">
      <property name="text">
       <string>10.0.1.2</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Frame Buffer Settings</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Width</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="widthSpinBox">
      <property name="maximum">
       <number>4096</number>
      </property>
      <property name="value">
       <number>1024</number>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Height</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="heightSpinBox">
      <property name="maximum">
       <number>4096</number>
      </property>
      <property name="value">
       <number>600</number>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>RGB Format</string>
      </property>
     </widget>
     <widget class="QComboBox" name="rgbFormatComboBox">
      <item>
       <property name="text">
        <string>RGB16</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RGB30</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RGB32</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RGB444</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RGB555</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RGB666</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RGB888</string>
       </property>
      </item>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Output Folder</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="toolButtonLineEdit">
       <property name="text">
        <string>~/</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ScreenshotDeviceSelectionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ScreenshotDeviceSelectionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>44</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
